src/main/java/seedu/smartlib/logic/commands/BorrowCommand.java

```diff
@@ -20,16 +21,16 @@
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a record of a reader borrowing a book. "
             + "Note that a book cannot be borrowed to multiple readers at the same time.\n"
             + "Note that readers may only borrow up to " + SmartLib.QUOTA + " books.\n"
-            + "Parameters: " + PREFIX_BOOK + "<book name> " + PREFIX_READER + "<reader name>\n"
-            + "Example: " + COMMAND_WORD + " " + PREFIX_BOOK + "The Hobbit " + PREFIX_READER + "Alex Yeoh";
+            + "Parameters: " + PREFIX_BARCODE + "<barcode> " + PREFIX_READER + "<reader name>\n"
+            + "Example: " + COMMAND_WORD + " " + PREFIX_BARCODE + "1000000001 " + PREFIX_READER + "Alex Yeoh";
```

Comment:
Maybe you might want to change the sample barcode to one which is not contained in SampleDataUtil.java?

e.g. 1234567890